app/views/user.py

```diff
@@ -322,17 +405,17 @@ def post(self):
             return make_response(jsonify(response))
 
 
-user_api = Blueprint('resources.user', __name__)
+user_api = Blueprint('views.user', __name__)
 api = Api(user_api)
 api.add_resource(
     RegisterUser,
-    '/register_user',
-    endpoint='register_user'
+    '/register',
+    endpoint='register'
 )
```

Comment:
code in lines 410-414 can be gracefully handle in one line for better code organisation